src/store/errors.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(Debug, Clone)]
+pub enum StoreError {
+    RepositoryNotFound(String),
+    PacBuildNotFound { name: String, repository: String },
+    RepositoryConflict(String),
+    PacBuildConflict { name: String, repository: String },
+    Unexpected(String),
+    Aggregate(Vec<StoreError>),
+}
```

Comment:
Did you look into [error-stack](https://crates.io/crates/error-stack) or [thiserror](https://crates.io/crates/thiserror)?